src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -458,12 +666,53 @@ class KinDynComputations {
                         const Vector6& baseAcc,
                         const VectorDynSize& s_ddot);
 
+    /**
+     * Return the frame acceleration, with the convention specified by getFrameVelocityRepresentation.
+     * (MatrixView and Span version)
+     *
+     * @warning As this method recomputes the accelerations of all links for each call, it may be computationally expensive.
+     *
+     * @warning the Span objects should point an already existing memory. Memory allocation and resizing cannot be achieved with this kind of objects.
+     * @return true if all went well, false otherwise.
+     */
+    bool getFrameAcc(const FrameIndex frameName,
+                     iDynTree::Span<const double> baseAcc,
+                     iDynTree::Span<const double> s_ddot,
+                     iDynTree::Span<double> frame_acceleration);
+
+    /**
+     * Compute the free floating jacobian for a given frame for the given representaiton.
+     *
+     * @return true if all went well, false otherwise.
+     */
     bool getFrameFreeFloatingJacobian(const std::string & frameName,
                                       iDynTree::MatrixDynSize & outJacobian);
 
+    /**
+     * Compute the free floating jacobian for a given frame for the given representaiton.
+     *
+     * @return true if all went well, false otherwise.
+     */
     bool getFrameFreeFloatingJacobian(const FrameIndex frameIndex,
                                       iDynTree::MatrixDynSize & outJacobian);
 
+    /**
+     * Compute the free floating jacobian for a given frame for the given representaiton (MatrixView implementation).
+     *
+     * @warning the MatrixView objects should point an already existing memory. Memory allocation and resizing cannot be achieved with this kind of objects.
+     * @return true if all went well, false otherwise.
+     */
+    bool getFrameFreeFloatingJacobian(const std::string & frameName,
```

Comment:
nitpick: you probably do not have to replicate both versions (with `std::string` and `FrameIndex`)?

Reply:
We wanted to have exact corresponding method between Span and non-Span, to simplify migration to the Span version.